src/main/java/org/apache/pirk/responder/wideskies/ResponderDriver.java

```diff
@@ -65,6 +66,13 @@ else if (SystemConfiguration.getProperty(ResponderProps.PLATFORM).equals("spark"
       ComputeResponse computeResponse = new ComputeResponse(fs);
       computeResponse.performQuery();
     }
+    else if (SystemConfiguration.getProperty(ResponderProps.PLATFORM).equals("storm"))
```

Comment:
we will need to abstract this out to implement this as a Strategy Pattern - the more backends we add like Flink, Spark, Beam and others in the future its only gonna get messy with multi - if-else stmts. 

For a fun context, let me point u to this code snippet from IBM SystemML, they r now adding Flink support which would be another another if-else block of code -  https://gist.github.com/smarthi/eb848e46621b7444924f